specs/spec_template.md

```diff
@@ -1,155 +1,203 @@
+    (https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/UIElement/ElementCursor.md)
+    which has some of the thinking about how this Xaml API relates to existing
+    Composition and WPF APIs. This is interesting background both for the current reader
+    and the future reader trying to understand why we designed it this way,
+    but not the kind of information
+    that would land on DMC.
 -->
 
-# Description
+# Conceptual pages (How To)
 
-<!-- TEMPLATE
-    Use this section to provide a brief description of the feature.
-    For an example, see the introduction to the PasswordBox control
-    (http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box).
--->
-
-# Examples
+_(This is conceptual documentation that will go to docs.microsoft.com "how to" page)_
 
 <!-- TEMPLATE
+    (Optional)
 
-    Use this section to explain the features of the API, showing
-    example code with each description. The general format is:
+    All APIs have a page on DMC, some APIs or groups of APIs have an additional high level,
+    conceptual page (called a "how-to" page). This section can be used for that content.
 
-    ## FirstFeatureName
+    For example, there are several Xaml controls for different forms of text input,
+    each with an API page, and then there's also a conceptual page that
+    discusses them collectively
+    (https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/text-controls).
 
-    Feature explanation text goes here, including why an app would use it, how it
-    replaces or supplements existing functionality.
+    Another way to use this section is as a draft of a blog post that introduces the new feature.
 
-    ```c#
-    void SampleMethod() {
-        var show = new AnExampleOf();
-        show.SomeMembers = AndWhyItMight(be, interesting)
-    }
-    ```
+    Sometimes it's difficult to decide if text belongs on a how-to page or an API page.
+    It's not important to make a final decision on that in this spec; we can always
+    adjust it when copying to DMC.
+-->
 
-    ## SecondFeatureName
+# API Pages
 
-    Feature explanation text goes here, including why an app would use it, how it
-    replaces or supplements existing functionality.
+_(Each of the following L2 sections correspond to a page that will be on docs.microsoft.com)_
 
-    ```c#
-    void SampleMethod() {
-        var show = new AnExampleOf();
-        show.SomeMembers = AndWhyItMight(be, interesting)
-    }
-    ```
+<!-- TEMPLATE
 
-    Code samples should be in C# and/or C++/WinRT.
+  Each of the L2 sections in this "API Pages" section corresponds to a page on DMC.
+
+  It's not necessary to have a section for every class member though:
+  * If its purpose and usage is obvious from it's name/type, it's not necessary to
+    create a section for it.
+  * If its purpose and usage is fully explained by brief description, either
+      put it in a table in the "Other [class] members" section
```

Comment:
The table from the lifecycle doc was super hard to read. Maybe add a note to the example of it that the sentences should be BRIEF and if it's more than, say, 150 characters it should get a subsection.